modules/apigee/README.md

```diff
@@ -0,0 +1,169 @@
+# Apigee
+
+This module simplifies the creation of a Apigee resources (organization, environment groups, environment group attachments, environments, instances and instance attachments). 
+
+## Example
+
+### All resources (CLOUD)
+
+```hcl
+module "apigee" {
+  source     = "./fabric/modules/apigee"
+  project_id = "my-project"
+  organization = {
+    display_name            = "My Organization"
+    description             = "My Organization"
+    authorized_network      = "my-vpc"
+    runtime_type            = "CLOUD"
+    billing_type            = "PAYG"
+    database_encryption_key = "123456789"
+    analytics_region        = "europe-west1"
+  }
+  envgroups = {
+    test = ["test.example.com"]
+    prod = ["prod.example.com"]
+  }
+  environments = {
+    apis-test = {
+      display_name    = "APIs test"
+      description     = "APIs Test"
+      deployment_type = "PROXY"
+      api_proxy_type  = "API_PROXY_TYPE_UNSPECIFIED"
+      envgroup        = "test"
+    }
+    apis-prod = {
+      display_name    = "APIs prod"
+      description     = "APIs prod"
+      deployment_type = "PROXY"
+      api_proxy_type  = "API_PROXY_TYPE_UNSPECIFIED"
+      envgroup        = "prod"
+    }
+  }
+  instances = {
+    instance-test-ew1 = {
+      region            = "europe-west1"
+      environments      = ["apis-test"]
+      psa_ip_cidr_range = "10.0.4.0/22"
+    }
+    instance-prod-ew1 = {
+      region            = "europe-west1"
+      environments      = ["apis-prod"]
+      psa_ip_cidr_range = "10.0.4.0/22"
+    }
+  }
+}
+# tftest modules=1 resources=11
+```
+
+### All resources (HYBRID)
```

Comment:
Do we need to clarify that this deploys the Apigee control plane only?

Reply:
done